packages/@okta/vuepress-theme-prose/global-components/EventHookEANote.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
Not sure if the warnings matter here or not. Seems like the content is rendering fine.

Reply:
Nice catch. This is an EA note that I have on another branch. It will display when I merge all the event hook filter branches into one.

Reply:
Actually, this is the note now that I look at it.